oqsprov/oqsprov_capabilities.c

```diff
@@ -613,29 +613,47 @@ static int oqs_group_capability(OSSL_CALLBACK *cb, void *arg)
                 OSSL_PARAM_END                                                \
         }
 
+#    if defined(OQS_ENABLE_SIG_dilithium_2)                 \
+        || defined(OQS_ENABLE_SIG_dilithium_3)              \
+        || defined(OQS_ENABLE_SIG_dilithium_5)              \
+        || defined(OQS_ENABLE_SIG_ml_dsa_44)                \
+        || defined(OQS_ENABLE_SIG_ml_dsa_65)                \
+        || defined(OQS_ENABLE_SIG_ml_dsa_87)                \
+        || defined(OQS_ENABLE_SIG_falcon_512)               \
+        || defined(OQS_ENABLE_SIG_falcon_padded_512)        \
+        || defined(OQS_ENABLE_SIG_falcon_1024)              \
+        || defined(OQS_ENABLE_SIG_falcon_padded_1024)       \
+        || defined(OQS_ENABLE_SIG_sphincs_sha2_128f_simple) \
+        || defined(OQS_ENABLE_SIG_sphincs_sha2_128s_simple) \
+        || defined(OQS_ENABLE_SIG_sphincs_sha2_192f_simple) \
+        || defined(OQS_ENABLE_SIG_sphincs_shake_128f_simple)
+#        define OQS_SIGNATURES_USED
+#    endif
+
+#    ifdef OQS_SIGNATURES_USED
 static const OSSL_PARAM oqs_param_sigalg_list[][12] = {
 ///// OQS_TEMPLATE_FRAGMENT_SIGALG_NAMES_START
-#    ifdef OQS_ENABLE_SIG_dilithium_2
+#        ifdef OQS_ENABLE_SIG_dilithium_2
```

Comment:
All of these indentations make sense, but also need to be within their [respective code generator fragment](oqs-template/oqsprov/oqsprov_capabilities.c/sigalg_names.fragment) to still exist after [the code generator](https://github.com/open-quantum-safe/oqs-provider/blob/main/CONFIGURE.md#pre-build-configuration) has run.